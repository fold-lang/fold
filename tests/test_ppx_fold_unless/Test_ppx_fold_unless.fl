
let _ = {
  //let x = 1;

  //unless! {
  //  (x <= 0) { print_endline "ok" }
  //};
};