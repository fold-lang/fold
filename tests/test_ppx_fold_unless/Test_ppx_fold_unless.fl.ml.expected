let () =
  let x = 1 in
  [%fl.macro_call
    Shape
      ("unless",
        [Seq (None, [Ident (Lower "x"); Sym "<="; Const (Int 0)]);
        Scope
          ("{",
            (Seq (None, [Ident (Lower "print_endline"); Const (String "ok")])),
            "}")])]
